redwood-testing/src/commonTest/kotlin/app/cash/redwood/testing/ViewTreesTest.kt

```diff
@@ -142,4 +145,20 @@ class ViewTreesTest {
 
     assertThat(widgetContainer.map { it.value }).isEqualTo(snapshot)
   }
+
+  @Test fun uiConfigurationWorks() = runTest {
+    TestSchemaTester {
```

Comment:
I think I want to pipe the ability to specify the initial `UiConfiguration` instance through to this generated function. But I'll do that as a follow-up.